lib/authHandlers.js

```diff
@@ -7,6 +7,10 @@ var sessions       = require('client-sessions'),
     bodyParser     = require('body-parser'),
     config         = require('./config'),
     parseDuration  = require('parse-duration');
+    LRUMap         = require('lru_map');
+
+var secondarySessionTokens = new LRUMap.LRUMap(10);
```

Comment:
I would allow this to be initialized via an env var that defaults to 10.  Concerns here of what this would mean when used at scale.